spec/rails_helper.rb

```diff
@@ -42,6 +42,11 @@
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
+  # Clean up uploaded files for rspec
+  config.after do
+    FileUtils.rm_rf(Dir["#{Rails.root}/public/uploads/test"])
```

Comment:
it is good to clean up files after tests, but one more, you don't need to create user and other models with images in for specs 
https://github.com/IF-115-Ruby/OSBB/blob/733d2e7d68b98546028a095eb0c91c1410839272/spec/factories/users.rb#L6
https://github.com/IF-115-Ruby/OSBB/blob/733d2e7d68b98546028a095eb0c91c1410839272/spec/factories/news.rb#L8
if you need them for seeds, you need just to specify that fields in seeds.rb file